support "local" node

Hi,

firstly, thanks for this plugin! We would like to use this plugin in our CI pipeline in which we use Alpine Container Images. Unfortunately the currently available Node 8 & 10 binaries doesn't work in Alpine. To use the plugin in Alpine, we have to replace the plugin node binaries with the Alpine node binaries (installed with `apk add nodejs`).
```
[ERROR] Failed to execute goal com.hubspot.maven.plugins:prettier-maven-plugin:0.10:check (default-cli) on project *******: Error trying to run prettier-java: Cannot run program "/home/maven/.m2/repository/com/hubspot/maven/plugins/prettier-maven-plugin/0.10/prettier-maven-plugin-0.10-node-12.13.0-linux.exe" (in directory "/"): error=2, No such file or directory -> [Help 1]
```

Instead of adding the Alpine binaries to the project, we thought about supporting a local node. What do you think about to introduce new cli option like `prettier.nodePath` where?

--- a/prettier-maven-plugin/src/main/java/com/hubspot/maven/plugins/prettier/PrettierArgs.java
+++ b/prettier-maven-plugin/src/main/java/com/hubspot/maven/plugins/prettier/PrettierArgs.java
@@ -99,6 +99,10 @@ public abstract class PrettierArgs extends AbstractMojo {
   private RepositorySystem repositorySystem;
 
   protected Path resolveNodeExecutable() throws MojoExecutionException {
+    if (nodePath != null && !nodePath.isEmpty()) {
+      getLog().info("Using customized nodePath: " + nodePath);
+      return Paths.get(nodePath);
+    }
     Artifact nodeArtifact = new DefaultArtifact(
         pluginDescriptor.getGroupId(),
         pluginDescriptor.getArtifactId(),
@@ -111,12 +115,7 @@ public abstract class PrettierArgs extends AbstractMojo {
       getLog().debug("Resolving node artifact " + nodeArtifact);
     }
 
-    File nodeExecutable = null;
-    if(nodePath != null && !nodePath.equals("")) {
-      nodeExecutable = new File(nodePath);
-    } else {
-      nodeExecutable = resolve(nodeArtifact).getFile();
-    }
+    File nodeExecutable = resolve(nodeArtifact).getFile();
     if (!nodeExecutable.setExecutable(true, false)) {
       throw new MojoExecutionException(
           "Unable to make file executable " + nodeExecutable

--- a/prettier-maven-plugin/src/main/java/com/hubspot/maven/plugins/prettier/PrettierArgs.java
+++ b/prettier-maven-plugin/src/main/java/com/hubspot/maven/plugins/prettier/PrettierArgs.java
@@ -51,6 +51,9 @@ public abstract class PrettierArgs extends AbstractMojo {
   @Parameter(defaultValue = "12.13.0", property = "prettier.nodeVersion")
   private String nodeVersion;
 
+  @Parameter(defaultValue = "", property = "prettier.nodePath")
+  private String nodePath;
+
   @Parameter(defaultValue = "0.7.0", property = "prettier.prettierJavaVersion")
   private String prettierJavaVersion;
 
@@ -108,7 +111,12 @@ public abstract class PrettierArgs extends AbstractMojo {
       getLog().debug("Resolving node artifact " + nodeArtifact);
     }
 
-    File nodeExecutable = resolve(nodeArtifact).getFile();
+    File nodeExecutable = null;
+    if(nodePath != null && !nodePath.equals("")) {
+      nodeExecutable = new File(nodePath);
+    } else {
+      nodeExecutable = resolve(nodeArtifact).getFile();
+    }
     if (!nodeExecutable.setExecutable(true, false)) {
       throw new MojoExecutionException(
           "Unable to make file executable " + nodeExecutable

